README.md

```diff
@@ -11,23 +14,38 @@
 [![Slack](https://img.shields.io/badge/Slack-@layer5.svg?logo=slack)](http://slack.layer5.io)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3564/badge)](https://bestpractices.coreinfrastructure.org/projects/3564)
 
-# layer5io/homebrew-tap
-Homebrew Tap for for installing the Meshery command line client: `mesheryctl`
+<div>&nbsp;</div>
+
+# Meshery/homebrew-tap
```

Comment:
nit

```suggestion
# meshery/homebrew-tap
```

Reply:
@nishantwrp do this need more changes??

Reply:
@manikmmalhotra i think you missed this comment https://github.com/meshery/homebrew-tap/pull/31#discussion_r919453551

Reply:
@nishantwrp it's done